test/run-tests.sh

```diff
@@ -94,4 +94,4 @@ vmTestNixExpr() {
 EOF
 }
 
-eval "${@:-run}"
```

Comment:
Top of that file still says:
```
#   Run test
#   ./run-tests.sh
```
But perhaps makes more sense to run the tests by default because that's what the name of the script suggests?

Reply:
It still runs the test by default, just via nix-build. This way the test runs in a sandbox and the result is cached so that tests for identical builds aren't re-run.

Reply:
Oh I see, neat